TradeItIosTicketSDK2/TradeItLinkedBrokerManager.swift

```diff
@@ -225,7 +225,7 @@ import PromiseKit
     }
     
     @objc public func getAllDisplayableLinkedBrokers() -> [TradeItLinkedBroker] {
-        return self.linkedBrokers.filter { $0.getEnabledAccounts().count > 0 || $0.isAccountLinkDelayedError}
+        return self.linkedBrokers.filter { $0.getEnabledAccounts().count > 0 || $0.isAccountLinkDelayedError || $0.error?.errorCode == TradeItErrorCode.sessionError }
```

Comment:
Maybe we can just use the `requiresAuthentication` method?
```
return self.linkedBrokers.filter { $0.getEnabledAccounts().count > 0 || $0.error?.requiresAuthentication() }
```

It contains in addition, the "brokerAccountError": https://github.com/tradingticket/TradeItIosTicketSDK2/blob/ed964cf9730a81a31f98f055a2ec341afb4dad89/TradeItIosTicketSDK2/TradeItErrorResult%2BErrorCodeHelpers.swift#L81

Reply:
Good call.